orientdb/src/main/java/com/yahoo/ycsb/db/OrientDBClient.java

```diff
@@ -71,7 +71,8 @@
   private static final String ORIENTDB_DOCUMENT_TYPE = "document";
 
   /**
-   * Initialize any state for this DB. Called once per DB instance; there is one DB instance per client thread.
+   * Initialize any state for this DB. Called once per DB instance; there is one
```

Comment:
I think these are all just the comments from the DB interface. Can we just leave them out and have the javadoc get inherited?